public/pages/Administration/components/webhook/WebhookProperties.scss

```diff
@@ -0,0 +1,32 @@
+@import "~@fider/assets/styles/variables.scss";
+
+.c-webhook-properties {
+  border-spacing: 0;
+
+  th {
+    padding: sizing(3);
+  }
+
+  td {
+    border-top: 1px solid get("colors.gray.100");
+    padding: sizing(2);
+  }
+
+  &__prop {
+    font-family: $font-code;
```

Comment:
there's a `get("font.family.code")`

Reply:
I searched it at the moment, but didn't spotted it, I'll switch to it